forms-flow-api/src/formsflow_api/resources/employeeData.py

```diff
@@ -37,3 +37,23 @@ def get():
             return err.error, err.status_code
         
         return userData, HTTPStatus.OK
+
+
+@cors_preflight("GET, OPTIONS")
+@API.route("/names", methods=["GET", "OPTIONS"])
+class EmployeeNames(Resource):
+    """Resource for managing employee names"""
+
+    @staticmethod
+    @profiletime
+    @auth.require
+    def get():
+        """Get employee names from ODS."""
+        args = request.args
```

Comment:
I'm wondering if we can restrict this to users with the `employee-search` role? Just so we can restrict this while testing it. An example of this can be found in `resources/application.py` -> `auth.has_role([REVIEWER_GROUP])`